Location.md

```diff
@@ -20,6 +20,32 @@ See HTML.
 
 See HTML.
 
+## Location.toString ( )
+
+1. Let _hrefDesc_ be DefaultInternalMethod([[GetOwnProperty]\], this, "href").
+
+1. Return the result of calling _hrefDesc_.[[Get]\].
+
+The property has the attributes { [[Writable]]: false, [[Enumerable]]: true, [[Configurable]]: false }.
+
+## Location.toJSON
+
+The initial value of Location.toJSON is undefined.
+
+The property has the attributes { [[Writable]]: false, [[Enumerable]]: true, [[Configurable]]: false }.
+
+## Location.valueOf ( )
+
+1. Return this.
```

Comment:
My bad, thanks. I see now the important part of these sections is their property descriptors, not their contents.